src/main/java/com/github/manolo8/darkbot/core/objects/facades/DiminishQuestFacade.java

```diff
@@ -0,0 +1,56 @@
+package com.github.manolo8.darkbot.core.objects.facades;
+
+import com.github.manolo8.darkbot.Main;
+import com.github.manolo8.darkbot.core.itf.Updatable;
+import com.github.manolo8.darkbot.core.objects.Gui;
+import eu.darkbot.api.API;
+import eu.darkbot.api.managers.QuestAPI;
+
+import static com.github.manolo8.darkbot.Main.API;
+
+public class DiminishQuestFacade extends Updatable implements API.Singleton {
+    private State state = State.NOT_EXIST;
+    private Gui gui;
+    private final QuestProxy.Quest diminishQuest = new QuestProxy.Quest();
+    
+    @Override
+    public void update() {
+        if (address == 0) state = State.NOT_EXIST;
+        else if (!"diminish_quests".equals(API.readString(address, 0x38))) {
+            address = 0;
+            state = State.NOT_EXIST;
+        }
+        else try {
+            gui = Main.INSTANCE.guiManager.getGui("diminish_quests");
+            state = State.values()[API.readInt(address, 0x50, 0x40) + 1];
+            diminishQuest.update(API.readAtom(address, 0x50, 0x60));
+        } catch (Exception ignored) {
```

Comment:
I using this implementation ~250h and I catch behavior where fields have incorrect values or this mediator was excluded from mediator flashmap and address referenced to unexpected place when quest not active. This code covered this cases but now exist case where all fields say that quest ready to accept like accepting timer counting, gui window showing, it have correct position etc, but in-game it look like quest not present. I'm solving it via accepting attempts timer that lock stitching to temp module on 30 min.

This catch-all should fused sensitivity logic that can interrupt active module in case when we cannot read correctly diminish quest state.

Reply:
Added forwarding catched exception.

Reply:
You can simply check index of the state instead of catching the exceptions.

Reply:
Done